model/f18_lat.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+from scipy.io import loadmat
+
+
+class f18_lat:
+    def __init__(
+            self,
+            x0=None,
+            x_ref=None,
+            C=np.diag([1, 1, 1, 1]),
+            Q=np.diag([1, 1, 1, 1]),
+            R=np.diag([1e6, 1e6]),
+            Qa=np.diag([100, 10, 10, 10, 10, 10, 10, 10]),
+            Ra=np.diag([1e6, 1e6])
+    ):
+        # initial x_ref setting from trim condition
+        self.x_trim = loadmat('../dat/f18_lin_data.mat')['x_trim_lon'].squeeze()
+        # x_ref default value : x_trim
+        if x_ref is not None:
+            self.x_ref = x_ref
+        else:
+            self.x_ref = self.x_trim
+        noise = np.array([np.random.normal(0, 0.1 * self.x_trim[0]),
+                          np.random.normal(0, 0.1 * self.x_trim[1]),
+                          np.random.normal(0, 1e-3),
+                          np.random.normal(0, 1e-3)])
+        # x0 default value : x_trim + noise
+        if x0 is not None:
+            self.x0 = x0
+        else:
+            self.x0 = self.x_trim + noise
```

Comment:
실제로 주고자 하는 reference는 `trim`에서 조금 벗어난 값이지만 선형화된 시스템의 경우 trim 점을 원점으로 하는 시스템이 됩니다.
따라서 아래와 같이 수정하는 것이 더 편한 방법이 되겠어요.
- `self.x_trim` 값은 결과를 plot할 때에만 값을 사용하기
```suggestion
        noise = np.array([np.random.normal(0, 0.1 * self.x_trim[0]),
                          np.random.normal(0, 0.1 * self.x_trim[1]),
                          np.random.normal(0, 1e-3),
                          np.random.normal(0, 1e-3)])
        # x0 default value : x_trim + noise
        if x0 is not None:
            self.x0 = x0
        else:
            self.x0 = noise
            
        if x_ref is not None:
            self.x_ref = x_ref
        else:
            self.x_ref = np.zeros(self.x0.shape)
```